### DISCUSSION TITLE
QUIC support

### DISCUSSION THREAD

**Discussion by LexterS999** [jinwyp/one_click_script#86] [Ideas]
What about adding QUIC support to V2ray/Xray as plugin? It's officially committed by IETF

**Comment by jinwyp** [comment]
Now you can choose 11 or 12 to install QUIC

**Comment by aanfath** [comment]
Amazing🤩

**Comment by aanfath** [comment]
> What about adding QUIC support to V2ray/Xray as plugin? It's officially committed by IETF

Ya it's good... But ask your near seated..
